config/initializers/typed_params.rb

```diff
@@ -24,3 +24,11 @@
     }
   },
 )
+
+TypedParams.types.register(:uuid,
+  archetype: :string,
```

Comment:
I don't really like that this has an archetype, but [`typed_params` currently appends new types](https://github.com/keygen-sh/typed_params/blob/7d95eb2d7a6e311158565c9931191e6b24bd4de1/lib/typed_params/types.rb#L16), so matching against a type that is also another type will always match the former type (in this case, string). Maybe we should add a `:prepend` option to `#register`, or add a `#prepend` method?

Either way, not gonna lose sleep over it. But may be something to eventually look at.